src/Microsoft.AspNet.Mvc.Core/FilterActionInvoker.cs

```diff
@@ -219,7 +368,36 @@ private IFilter[] GetFilters()
         private async Task InvokeAllActionFiltersAsync()
         {
             _cursor.SetStage(FilterStage.ActionFilters);
+
+            Debug.Assert(_resourceExecutingContext != null);
+            var bindingContext = new ActionBindingContext()
```

Comment:
We could just put the ABC on the ResourceFilterContext, but that has some subtle differences.  I really don't think the copying is bad, it's 5-6 properties, and that's kinda what DTOs do. Let's discuss.